### PR TITLE
chore:  add test to notification.service

### PR DESCRIPTION
**Pull request by yannick-aneo**

### CODE REVIEW

File: src/app/services/notification.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  });
+  
+  it('should open a snackBar with success status', () => {
+    const status = (notificationStatus: NotificationStatus) => {
+      return {
+        duration: 5000,
+        horizontalPosition: 'end',
+        panelClass: notificationStatus
+      }; 
+    };
```
Comment (Faust1-2was-Aneo) [inline src/app/services/notification.service.spec.ts:33]: Why creating a function for open ? Another solution could have been:
```suggestion
    const status = {
      duration: 5000,
      horizontalPosition: 'end',
      panelClass: 'success'
    }; 
```

File: src/app/services/notification.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    const spy = jest.spyOn(snackBar, 'open');
+    service.success('succes');
+    expect(spy).toHaveBeenCalledWith('succes', 'Close', status('success'));
```
Comment (Faust1-2was-Aneo) [inline src/app/services/notification.service.spec.ts:36]:
```suggestion
    expect(spy).toHaveBeenCalledWith('succes', 'Close', status);
```

File: src/app/services/notification.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    const status = (notificationStatus: NotificationStatus) => {
+      return {
+        duration: 5000,
+        horizontalPosition: 'end',
+        panelClass: notificationStatus
+      }; 
+    };
+    const spy = jest.spyOn(snackBar, 'open');
+    service.warning('attention');
+    expect(spy).toHaveBeenCalledWith('attention', 'Close', status('warning'));
```
Comment (Faust1-2was-Aneo) [inline src/app/services/notification.service.spec.ts:49]: Same here
```suggestion
    const status = {
      duration: 5000,
      horizontalPosition: 'end',
      panelClass: 'warning'
    };
    const spy = jest.spyOn(snackBar, 'open');
    service.warning('attention');
    expect(spy).toHaveBeenCalledWith('attention', 'Close', 'status');
```

File: src/app/services/notification.service.spec.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    const status = (notificationStatus: NotificationStatus) => {
+      return {
+        duration: 5000,
+        horizontalPosition: 'end',
+        panelClass: notificationStatus
+      }; 
+    };
+    const spy = jest.spyOn(snackBar, 'open');
+    service.error('erreur');
+    expect(spy).toHaveBeenCalledWith('erreur', 'Close', status('error'));
```
Comment (Faust1-2was-Aneo) [inline src/app/services/notification.service.spec.ts:62]: Same here:
```suggestion
    const status = {
      duration: 5000,
      horizontalPosition: 'end',
      panelClass: notificationStatus
    };
    const spy = jest.spyOn(snackBar, 'open');
    service.error('erreur');
    expect(spy).toHaveBeenCalledWith('erreur', 'Close', status('error'));
```

**Review by Faust1-2was-Aneo — CHANGES_REQUESTED** [review]

File: src/app/services/notification.service.spec.ts (hunk 1 above)
Comment (yannick-aneo) [reply src/app/services/notification.service.spec.ts:33]: It was for avoiding to create three objects for each status.

**Review by Faust1-2was-Aneo — APPROVED** [review]

**Pull request merged**